web/tests/search_suggestion.test.js

```diff
@@ -516,7 +514,7 @@ test("check_is_suggestions", ({override, mock_template}) => {
     assert.equal(describe("-is:dm"), "Exclude direct messages");
     assert.equal(describe("-is:starred"), "Exclude starred messages");
     assert.equal(describe("-is:mentioned"), "Exclude @-mentions");
-    assert.equal(describe("-is:alerted"), "Exclude alerted messages");
+    assert.equal(describe("-is:watched"), "Exclude messages with watched phrases");
```

Comment:
Currently renamed "Exclude alerted messages" to "Exclude messages with watched phrases". Need suggestions on naming this.